### DISCUSSION TITLE
where do I set WINEARCH when creating a wrapper?

### DISCUSSION THREAD

**Discussion by fqx** [The-Wineskin-Project/WineskinServer#450] [Q&A]
I have a exe that have to set WINEARCH=win32

**Comment by Rastafabisch** [comment]
You could just use a 32bit engine. If I recall correctly the last 32bit wine engine is WS11CX21.2 (or similar).

**Reply by fqx** [reply]
please see below https://github.com/Gcenx/WineskinServer/discussions/450#discussioncomment-6814222

**Comment by Gcenx** [comment]
The 32Bit only engines are no longer supported on WineCX22.1.1-x & WineCX23.0.0 base engines will be supported going forward.

The legacy engines were all removed from the EngineList as these are incompatible with macOS Ventura and/or macOS Sonoma.

**Reply by fqx** [reply]
@Gcenx So, is there a way to set WINEARCH=win32 to x64 engine?

**Reply by Gcenx** [reply]
This env doesn't work in the newer crossover-sources versions, even when `WINEARCH=win32` is set the `WINEPREFIX` won't generate as 32Bit still 64Bit.

There's also very few cases where an actual 32Bit prefix is required, 99% of applications/games will work just fine with the current none 32Bit only Engines.
